rosidl_adapter_proto/resource/msg.proto.em

```diff
@@ -38,7 +38,7 @@ for annotation in message.structure.annotations:
             and ("language" in annotation.value) \
             and (annotation.value["language"] == "comment") \
             and ("text" in annotation.value):
-        comment = "//" + re.sub(r"\\n", "\n//", annotation.value["text"])
```

Comment:
I haven't tried it. But are you sure, it actually works like that?

Old code: `r"\\n"`
New code: `"\n"`

So you removed the `r` and one of the `\`, which basically modified `"\\\\n"` to `"\n"`

Reply:
Yea, I didn't mean if the code works on a custom crafted example string. I was more talking about an actual .msg / .idl file:

```
    @verbatim (language="comment", text=
      "This was originally provided as an example message." "\n"
      "It is deprecated as of Foxy" "\n"
      "It is recommended to create your own semantically meaningful message." "\n"
      "However if you would like to continue using this please use the equivalent in example_msgs.")
```

You can see that there are a lot of textual `"\n"` strings in the comment. And I was unsure if those appear later in python as literal or interpreted strings, so if those are interpreted just as a line-feed (0xLF / `"\n"`) or if they appear as backslash+n (0x5c, 0x6e / `"\\n"`).

But if you tried an idl file with actual multiline comments and your code works, then it must be interpreted as actual line-feed.

Reply:
I see your point. I had the same doubts. I tried an IDL file with actual multiline comments, and they seem to be interpreted as actual line-feed. At least in my environment.